Storage/SQL/SQLiteHistory.swift

```diff
@@ -451,46 +451,28 @@ extension SQLiteHistory: BrowserHistory {
             args = []
         }
 
-        let ungroupedSQL = [
-        "SELECT \(TableHistory).id AS historyID, \(TableHistory).url AS url, title, guid, domain_id, domain,",
-        "COALESCE(max(case \(TableVisits).is_local when 1 then \(TableVisits).date else 0 end), 0) AS localVisitDate,",
-        "COALESCE(max(case \(TableVisits).is_local when 0 then \(TableVisits).date else 0 end), 0) AS remoteVisitDate,",
-        "COALESCE(count(\(TableVisits).is_local), 0) AS visitCount",
-        "FROM \(TableHistory)",
-        "INNER JOIN \(TableDomains) ON \(TableDomains).id = \(TableHistory).domain_id",
-        "INNER JOIN \(TableVisits) ON \(TableVisits).siteID = \(TableHistory).id",
+        let sql = [
+        "SELECT",
+            "history.id AS historyID, history.url, title, guid, domain_id, domain,",
```

Comment:
If you're going to eliminate `TableHistory` and friends in favor of inlining `"history"`, be consistent, and don't do it as part of this commit. (Though I appreciate that the change makes the diff easier to read!)